.github/workflows/CI.yml

```
@@ -50,6 +50,13 @@ jobs:
             ${{ runner.os }}-test-
             ${{ runner.os }}-
       - uses: julia-actions/julia-buildpkg@latest
+      - name: "Compat fix for Julia < v1.5"
+        if: ${{ (matrix.version == '1.3') || (matrix.version == '1.0') }}
+        run: |
+          # Remove Minio from the test project
+          write("Project.toml", read(`grep -v '^Minio =' Project.toml`, String))
+          run(`sed -i -e 's/, "Minio"//' Project.toml`)
+        shell: julia --project=. --startup=no --color=yes {0}
```

I just realized the scope of the madness going on here.
```suggestion
        run: |
          # Remove Minio from the test project
          sed -i -e '/^Minio =/d; s/"Minio"\(, \)\{0,1\}//' Project.toml
```
Using `sed` for everything here and cutting out the Julia middle man. One small tweak I did is optionally cut out the comma in case the order is changed

---

Sounds good 😄

---

Round two. The case I originally coded for would handle if "Minio" was in any position in the array except for the last element. This corrects that and also supports "Minio" being the only element in the array (which should never be the case)
```suggestion
        run: |
          # Remove Minio from the test project
          sed -i -e '/^Minio =/d; s/\(, \)"Minio"//; s/"Minio"\(, \)\{0,1\}//' Project.toml
```